README.md

```diff
@@ -2411,6 +2415,7 @@ IEEE Trans. Pattern Anal. Mach. Intell., 45(1)785:795, 2022. [Full PDF incl. Sup
 - <a name="Wang22icra"></a>Wang, Z., Yuan, D., Ng, Y., Mahony, R.,   
 *[A Linear Comb Filter for Event Flicker Removal](https://doi.org/10.1109/ICRA46639.2022.9812003)*,  
 IEEE Int. Conf. Robotics and Automation (ICRA), 2022. [PDF](https://arxiv.org/pdf/2205.08090.pdf), [Code and Data](https://github.com/ziweiWWANG/EFR), [Youtube](https://www.youtube.com/watch?v=1XgOknDIV00).
+- [Guillen-Garcia, J., Palacios-Alonso, D., Cabello, E., Conde, C., IEEE Access, 2022](#Guillen-Garcia22).
```

Comment:
This does not look like a paper on event denoising... Please reconsider this link.

Reply:
The article introduces an M-out-of-N event declaration filter. Maybe I should add something like "M-out-of-N Event Declaration Filter for Improved Event Detection"?

Reply:
what's the difference with a background activity filter?

Reply:
The BAF presented by Dellbruck addresses the issue by eliminating events not supported by recent activity in the nearby periphery. The M-out-of-N filter is a generalization.

![image](https://github.com/user-attachments/assets/5c5704a9-bda3-42b4-9dc8-f0cf1b4fcb07)

Reply:
So we get better results:

![image](https://github.com/user-attachments/assets/80129ab0-28ef-4421-85a4-3fdaf346faef)

Reply:
![image](https://github.com/user-attachments/assets/783348b6-e897-4f54-ac9c-76b0ae561043)

Reply:
![image](https://github.com/user-attachments/assets/bc02e052-daa3-4605-8ca8-4f8c798e4be4)

Reply:
> The article introduces an M-out-of-N event declaration filter. Maybe I should add something like "M-out-of-N Event Declaration Filter for Improved Event Detection"?

yes, please

Reply:
Used this format:

- [Guillen-Garcia et al., IEEE Access, 2022](#Guillen-Garcia22).
*M-out-of-N Event Declaration Filter for Improved Event Detection (Unsupervised adaptive multi-object tracking-by-clustering algorithm with a bio-inspired system)*.